client/keys/show.go

```diff
@@ -71,10 +71,16 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		}
 	} else {
 		pks := make([]cryptotypes.PubKey, len(args))
-		for i, keyref := range args {
-			k, err := fetchKey(clientCtx.Keyring, keyref, clientCtx.AddressCodec)
+		keyFilter := make(map[string]struct{})
+		for i, keyRef := range args {
+			if _, ok := keyFilter[keyRef]; ok {
+				cmd.Printf("WARNING: duplicate keys found: %s.\n", keyRef)
```

Comment:
Do warnings go to stdout instead of stderr? If so, would be useful to show them on stderr

Reply:
@odeke-em Hi sir, I;m not sure which output should be used, as there is no consistency
![image](https://github.com/cosmos/cosmos-sdk/assets/150114626/31ca68de-140e-42fa-aaeb-38daac7b9475)

Reply:
Done